buildchow/todo.adoc

```diff
@@ -0,0 +1,69 @@
+= Advantages noticed within a week:
+
+* fixing JRE across all jobs
+* relasing target platform without breaking builds
```

Comment:
"Show a man there's a typo, and he'll fix it. Teach a man to fish (for typos) and he'll eat (typos) for days." - Some dead guy on a tree